Products/ZenUI3/browser/resources/js/zenoss/EventPanels.js

```diff
@@ -1498,6 +1498,11 @@
                     this.applyState(state);
                     this.filterRow.storeSearch();
                 } catch(e) { }
+            // for cases with not encoded parameters
+            } else if (qs) {
+                state = {"filters": Ext.urlDecode(qs)};
```

Comment:
This looks like an already existing issue, but `state` is never declared. This means by default it is being declared and defined as a global variable, which can introduce issues if some other code touches a `state` global variable as well.